api/server/agent.go

```diff
@@ -22,9 +23,12 @@ func (tapi ToDDApi) Agent(w http.ResponseWriter, r *http.Request) {
 	// Retrieve query values
 	values := r.URL.Query()
 
-	var agent_list []defs.AgentAdvert
-
-	var tdb = db.NewToddDB(tapi.cfg)
+	agent_list, err := tapi.tdb.GetAgents()
```

Comment:
My intention was not to populate agent_list here, but rather initialize it as empty. Can we remove this, or is there a reason you're pulling from GetAgents()?